Header header

#Температура в главном корпусе [С]
float64 temperature